core/lib/instrument_api.h

```diff
@@ -5052,6 +5052,7 @@ dr_insert_clean_call_ex(void *drcontext, instrlist_t *ilist, instr_t *where,
                         void *callee, dr_cleancall_save_t save_flags,
                         uint num_args, ...);
 
+DR_API
```

Comment:
If we want to export this it needs a cleaned-up doxygen comment to document it.  I don't think it was meant to be exported: it is only used internally for annotation handling, and should have stayed in instrument.h when I split off the exported routines.

It looks like you're using it for the test, and if we think it's useful for clients, sure, let's export it, but it needs to be documented, and it needs an entry in the changelist.